code/modules/mob/living/carbon/carbon.dm

```diff
@@ -668,6 +668,18 @@ var/list/ventcrawl_machinery = list(/obj/machinery/atmospherics/unary/vent_pump,
 	else
 		return initial(pixel_y)
 
+/mob/living/carbon/emp_act(severity)
+	for(var/obj/item/organ/internal/O in internal_organs)
+		O.emp_act(severity)
+	..()
+
+/mob/living/carbon/Stat()
+	if(statpanel("Status"))
+		var/obj/item/organ/internal/xenos/plasmavessel/vessel = get_int_organ(/obj/item/organ/internal/xenos/plasmavessel)
```

Comment:
i think i was commenting on this organ being snowflaked into carbon/Stat() instead of organs having a general stat reporting proc